src/components/List.js

```diff
@@ -1,5 +1,74 @@
 import React from 'react';
 
-const List = () => <h1>List</h1>;
+const List = () => (
```

Comment:
🤔 i thought this pr was only going to contain the style changes, no?

Reply:
I included  these list items to demonstrate that the app's navbar stays at the bottom of the viewport no matter what. I will remove them before this PR is merged